fastestimator/op/numpyop/meta/delete.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Any, Dict, Iterable, List, Union
+
+import numpy as np
+
+from fastestimator.op import NumpyOp
+
+
+class Delete(NumpyOp):
```

Comment:
not sure this should be a 'meta' op. This is more like univariate. Meta so far refers to Ops which take other Ops as input

Reply:
we should extend this definition of meta.    I also have BatchOp that I put in meta folder.

Reply:
Or create a new category?

Reply:
I think those are both examples of univariate. they take one or more keys and act on them each individually